circuitbreaker.py

```diff
@@ -57,7 +60,10 @@ def call(self, func, *args, **kwargs):
         :param func: Decorated function
         """
         if self.opened:
-            raise CircuitBreakerError(self)
+            if self.fallback_function:
+                self.fallback_function()
```

Comment:
Thanks for PR @duilio92 !
I think it makes sense, to pass the `*args` and `*kwargs` to the fallback function as well. What do you think?

Reply:
Yes you are right! I will change it as soon as posible.